src/main/java/seedu/address/logic/commands/HelpToDoCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.events.ui.ShowHelpRequestEvent;
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.ModelToDo;
+
+/**
+ * Format full help instructions for every toDoList command for display.
+ */
+public class HelpToDoCommand extends CommandToDo {
```

Comment:
we do not need this extra class, given that we already have a help command, as we expect that the help command will show the user help for commands relating to both the todolist and the calendar

Reply:
Agree